sys/cbor/cbor.c

```diff
@@ -92,6 +96,18 @@
 #endif
 
 #ifndef CBOR_NO_FLOAT
+
+/* pack to force aligned access on ARMv7 (buggy GCC) */
+#pragma GCC diagnostic error "-Wcast-align"
```

Comment:
However, after thinking about this a little bit, my conclusion is:
1.) I would rephrase my own comment from #2198 to: "pragmas that may lead to different output on different compilers should be avoided wherever possible".
2.) I don't quite understand which bug in GCC this refers to.
3.) The pragma would only add one more switch to GCC which IMO should not create a problem.